src/session.rs

```diff
@@ -76,30 +76,12 @@ pub fn start(config: &Config, initial_request: Option<String>) -> i32 {
 
                 let language_id = filetypes.get(&request.meta.filetype);
                 if language_id.is_none() {
-                    debug!(
+                    let msg = format!(
```

Comment:
can you add to the commit message a scenario that this commit improves?
It's late here and I think I forgot how to reproduce it.

Reply:
Added this and more context. For me it's mostly either rust-analyzer crashing or (maybe very rarely) the out-of-memory killer. I reproduce it by `pkill`ing the language server, though note that there's a small race condition where requests sent to the language server within a second of `pkill`ing it are ignored.